tests/ignite/base/test_mixins.py

```diff
@@ -1,7 +1,8 @@
 from ignite.base import Serializable
 
 
-def test_load_state_dict():
+def test_state_dict_and_load_state_dict():
 
     s = Serializable()
```

Comment:
I don't understand the test... more precisely I don't understand why this method is not abstract in the class or why it does not have a default implementation.

Reply:
I think we can treat it as a "smoke test" for `Serializable` :)
About method, as a [common practice](https://docs.python.org/3/library/exceptions.html#NotImplementedError) we can raise there `NotImplementedError`:

> This exception is derived from RuntimeError. In user defined base classes, abstract methods should raise this exception when they require derived classes to override the method, or while the class is being developed to indicate that the real implementation still needs to be added.

```python
    def state_dict(self) -> OrderedDict:
        raise NotImplementedError
```

Reply:
Or use `abstractmethod` from `abc`